.travis.yml

```diff
@@ -13,8 +13,14 @@ matrix:
         - php: 5.4
         - php: 5.5
         - php: 5.6
+        - php: 5.6
+          env: SYMFONY_VERSION="^2.1"
```

Comment:
what for?
if to test against lowest supported Symfony components, then it won't test `stopwatch 2.0` which we support, but 2.1.
for testing lowest, we have already 5.3 job with `--prefer-lowest`